src/Tenancy.php

```diff
@@ -31,6 +31,16 @@ class Tenancy
      */
     public function initialize($tenant): void
     {
+        if (! is_object($tenant)) {
+            $tenant = $tenant;
```

Comment:
This line it's not doing nothing and can be removed.

Reply:
Nice catch. So this all works now?